taskcluster/fenix_taskgraph/gradle.py

```diff
@@ -10,10 +10,30 @@
 from taskgraph.util.memoize import memoize
 
 
-@memoize
+
 def get_variant(build_type, engine):
-    output = _run_gradle_process('printVariant', variantBuildType=build_type, variantEngine=engine)
```

Comment:
This was the only case where `variantBuildType` and `variantEngine` were used.
If you aren't using them anymore, you can remove the associated logic from `app/build.gradle`

Reply:
Okay, I wasn't sure about the usage. Thanks for telling me. I removed the dead code!